cypress/e2e/registration.cy.js

```diff
@@ -1,11 +1,36 @@
 /// <reference types='cypress' />
 
-describe('Student Registration page', () => {
-  before(() => {
-
-  });
-
-  it('', () => {
-
+describe('Automation Practice Form', () => {
+  it('should fill all fields except picture and submit the form', () => {
+    cy.visit('https://demoqa.com/automation-practice-form');
```

Comment:
The URL of the page under test is hardcoded. It's better to put it in a configuration file or environment variable, so you can change it easily without modifying the test code.